mcpwm_foc.c

```diff
@@ -4088,8 +4116,8 @@ static void svm(float alpha, float beta, uint32_t PWMHalfPeriod,
 	// sector 1-2
 	case 1: {
 		// Vector on-times
-		uint32_t t1 = (alpha - ONE_BY_SQRT3 * beta) * PWMHalfPeriod;
-		uint32_t t2 = (TWO_BY_SQRT3 * beta) * PWMHalfPeriod;
+		uint32_t t1 = (alpha - beta13) * PWMHalfPeriod;
```

Comment:
I notice that all these values are truncated. I'm not sure if it's worth the performance penalty to add `0.5f` to the end to get a rounding-ish behavior.